web-console/future-openshift/diagnostics/diagnostics.md

```diff
@@ -0,0 +1,55 @@
+# Generating Diagnostic Report
+
+Users can download a cluster diagnostic report based on must-gather tool for upload to Red Hat Support or their own diagnostics.
+
+## Accessing 'Generate Report' actions
+
+There are several ways the user can access the `Generate (diagnostics) report` action.
+
+### Via the masthead help menu
+![Masthead help click](img/1-1a-clusterdash-startdownload.png)
+![Masthead help menu](img/1-1b-clusterdash-startdownload.png)
+
+### Via the cluster dashboard status card kebab menu
+![Cluster dashboard status card kebab click](img/1-2a-clusterdash-startdownload.png)
+![Cluster dashboard status card kebab menu](img/1-2b-clusterdash-startdownload.png)
+
+### Via a cluster dashboard status card message action
+![Cluster dashboard status card message click](img/5-1-clusterdash-CNVerror.png)
+- The cluster dashboard will soon support warning messages to notify the user that CNV or OCS are degraded. From these messages the user will be able to immediately `Generate report` using the action link within the message.
+- This access point is unique in that when the dialog is launched, it will already be scoped to the related product for which the message it was launched from is related to. In this shown example case, only `Container-native Virtualization` would be checked by default.
+
+## 'Generate Diagnostic Report' dialog
+![Generate Diagnostic Report dialog select](img/2-1-reportdialog.png)
+![Generate Diagnostic Report dialog select](img/2-2-reportdialog-menu.png)
+- The user can select what products and what time period of information should be included in the report and `Generate` it. The default state when launched from masthead help or status card kebab would be all 3 products selected and 8 hour time period.
```

Comment:
```suggestion
- The user can select which components and what time period of information should be included in the report and `Generate` it. The default state when launched from masthead help or status card kebab would be all components selected over an 8 hour time period.
```

Referring to these as components rather than products probably makes sense. In the future some of the options in here may be freely available non-product operators installed from OperatorHub, rather than products.